src/main/java/net/mcreator/ui/modgui/BlockGUI.java

```diff
@@ -1367,6 +1326,8 @@ else if (page == 6) {
 		if (model != null && model.getType() != null && model.getReadableName() != null)
 			renderType.setSelectedItem(model);
 
+		disableOffset.setEnabled(allowCustomBoundingBox());

```

Comment:
Is this part needed. I think the enabled/disabled state is already controlled by the block base combo box listener. Or did it not work?

Reply:
Tested it, the offset box can be removed. For some reasons the bounding box list isn't disabled correctly without this part though

Reply:
I have tried commenting out both and then opened with leaves block base for test and it opened disabled. Or is there some other case where it does not work?

Reply:
I did the same, and while it doesn't let me add new boxes, I can still edit existing ones

Reply:
Hmm, I will check